frontend/src/bundles/auth/pages/styles.module.scss

```diff
@@ -1,3 +1,5 @@
+@import "@assets/utils"; /* stylelint-disable-line scss/at-import-partial-extension, import-notation */
```

Comment:
```suggestion
@import "@assets/utils";
```

Reply:
Resolved in [6579ba9](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/95/commits/6579ba9171ee8abbe610455f37765f4a2dc4e37e)